lib/Doctrine/Sequence/Db2.php

```diff
@@ -80,7 +80,7 @@ public function currId($sequenceName)
              . $this->conn->quoteIdentifier($this->conn->formatter->getSequenceName($sequenceName))
              . ' AS VAL FROM SYSIBM.SYSDUMMY1';
 
-        $stmt   = $this->conn->query($sql);
+        $stmt   = $this->conn->standaloneQuery($sql);
```

Comment:
I had changed this from `$db->query` in the previous PR to `$this->conn->query` which isn't actually quite right, since `$this->conn->query` expects DQL. This is being sent an SQL string, which is what `standaloneQuery` is for (also the result of `$this->conn->query` doesn't have a `fetchAll` method, which is called below).